### ISSUE TITLE
Bug in `doing issue close`

### ISSUE THREAD

**Issue by timvink** [ing-bank/doing-cli#86]
Getting this error, something to dive into

![image](https://user-images.githubusercontent.com/5570380/132217123-41c4d522-d4f3-4eeb-be3b-77fa7f292c93.png)

**Comment by timvink** [comment]
I suspect I understand what's going on:

- ING has a new devops policy that states every feature should have story points assigned
- That policy is checked when updating issues as well, including (`az boards work-item update`). You can see this error in the UI as well, when changing status to closed:

![image](https://user-images.githubusercontent.com/5570380/132221538-cb82af38-cb2e-4d8a-9e67-c9f647a47a2c.png)

I've also noticed this policy is *not* enforced when creating issues.

We've already added support for storypoints in https://github.com/ing-bank/doing-cli/pull/65. `doing-cli` should work for any organization using any kind of policies / custom workflow. I see some possible 'solutions':

1) We catch this specific error and add to the error message to bulk update open issues without any story points. We can add a 'how to' guide in the docs on how to do this.
2) We add a custom parameter to `doing issue close` to specify the number of storypoints to update the issue with. Using that, you can close issue that didn't have any storypoints assigned yet. 
3) We could go further and make this a setting in the global config (if an issue does not have any storypoints when closing, then assign x before closing).

**Comment by tijlk** [comment]
I'd like to vote for 'solution' 3.

**Comment by operte** [comment]
Solution 3 feels to me like working around company policies. Wouldn't we be hindering the usage of storypoints as a good planning practice? 

On the other hand I can understand that some teams use other tools for planning. In those cases, some team members might create work items on the browser, forget to add storypoints, and then when a `doing-cli` user tries to close such a work item they will have their workflow disrupted.
In those cases, this feature could be quite useful.

**Comment by tijlk** [comment]
It should not be a default, but you should be able to set a default if the workflow of your team is currently not in line yet with the policies set out by your IT department.

That said... I just discovered, that you can already do this in doing-cli, by the following to your `.doing-cli-config.yml` file:
```
defaults:
    DOING_WORKON_STORY_POINTS: 1
```
That is sufficient for me for now.

**Comment by operte** [comment]
Right, but that option only works during `doing create`, which is great if you always use doing, but I guess some of your team members do not and they will forget to add story points to their work items. 

Let's work on this option for `doing close`. 
Tim already worked on providing an informative error message when the work item cannot be closed due to lack of story points. In addition to that message, we can inform the used that they can use an option on `doing close` (and even set a default option) to add a number of story points if this error occurs.

**Comment by timvink** [comment]
released in [v1.213])https://github.com/ing-bank/doing-cli/releases/tag/v1.2.13)

**Comment by timvink** [comment]
@tijlk don't forgot to update the story points defaults for `doing create issue` also:

```yaml
defaults:
    DOING_WORKON_STORY_POINTS: 1
    DOING_CREATE_ISSUE_STORY_POINTS: 1
```
